lib/ansible/parsing/vault/__init__.py

```diff
@@ -68,13 +68,8 @@
     from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
     from cryptography.hazmat.backends import default_backend
     HAS_PBKDF2HMAC = True
-except ImportError:
+except:
```

Comment:
don't use a catchall, if there are other errors we need to capture here please use specific exception captures